include/arch/x86/io.h

```diff
@@ -180,7 +180,7 @@ int dm_emulate_pio_post(struct vcpu *vcpu);
  *  @param value The 32 bit value to write.
  *  @param addr The memory address to write to.
  */
-static inline void mmio_write_long(uint32_t value, mmio_addr_t addr)
+static inline void mmio_write_long(uint32_t value, vaddr_t addr)
```

Comment:
Can we also redefine the mmio_addr_t type like vaddr_t so that the mmio_addr_t is used in io.h?
typedef void * mmio_addr_t